src/jsoncharutils_tables.h

```diff
@@ -201,8 +201,8 @@ const uint32_t digit_to_val32[886] = {
 // per instruction set).
 ///
 
-constexpr int FASTFLOAT_SMALLEST_POWER = -325;
-constexpr int FASTFLOAT_LARGEST_POWER = 308;
+constexpr int smallest_power = -344;
```

Comment:
I'm curious what leads to the choice of -344? Mind you, I don't know why we chose -325 before either :)

Reply:
I have adjusted the minimal power to be -342. Powers smaller than -342 would automatically lead to zero values.

```
/**
 * The smallest non-zero float (binary64) is 2^−1074.
 * We take as input numbers of the form w x 10^q where w < 2^64.
 * We have that w * 10^-343  <  2^(64-344) 5^-343 < 2^-1076.
 * However, we have that 
 * (2^64-1) * 10^-342 =  (2^64-1) * 2^-342 * 5^-342 > 2^−1074.
 * Thus it is possible for a number of the form w * 10^-342 where 
 * w is a 64-bit value to be a non-zero floating-point number.
 *********
 * Any number of form w * 10^309 where w>= 1 is going to be 
 * infinite in binary64 so we never need to worry about powers
 * of 5 greater than 308.
 */
```